bin/image-create-azure.sh

```diff
@@ -28,19 +28,19 @@ if [ -z "${TF_VARS_FILE}" ]; then
 fi
 
 # Get vars from tfvars-file
-arm_client_id=$(grep "client_id" "$TF_VARS_FILE" |
+arm_client_id=$(grep "^client_id" "$TF_VARS_FILE" |
```

Comment:
Isn't it possible to parse with the json2hcl tool? Or what is that for othewise?

Reply:
I left it as it is because I wanted it to work without Docker image somehow, but I think I changed my mind as well. I will change it for images in a later pull request.